test/dataViewObjectTest.ts

```diff
@@ -100,6 +100,18 @@ module powerbi.extensibility.utils.dataview.test {
 
                 expect(actualValue).toBe(defaultValue);
             });
+
```

Comment:
Yes, it does. The "should return false is the property defined as false" covers this case.